code/datums/wires/airalarm.dm

```diff
@@ -1,7 +1,7 @@
 /datum/wires/airalarm
 	holder_type = /obj/machinery/airalarm
 	proper_name = "Air Alarm"
-	req_knowledge = JOB_SKILL_MASTER
+	req_knowledge = 4
```

Comment:
Defines might be better, even if we have to do something like `SKILL_LEVEL_FOUR`
It's easier to link them together and reason about. Makes searches and mass-replacements easier.